### ISSUE TITLE
lore -olint's `check_texturl_matches_href` and `check_a_py_listing` methods invoke getAttribute wrong

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4115]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4115|
|Type|defect|
|Created|2009-11-17 16:17:52Z|
|Branch|https://github.com/twisted/twisted/tree/lore-lint-getattribute-fixes-4115|

These two methods call `getAttribute` with two arguments, but it accepts only one.

<details><summary>Searchable metadata</summary>

```
trac-id__4115 4115
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_lore_lint_getattribute_fixes_4115 branches/lore-lint-getattribute-fixes-4115
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__lore lore
keywords__ 
time__1258474672000000 1258474672000000
changetime__1258514528000000 1258514528000000
version__None None
owner__ 
cc__therve
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [27526]) Branching to 'lore-lint-getattribute-fixes-4115'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [27527]) Test the link text checker; delete the license checker

refs [#4115](https://github.com/twisted/twisted/issues/4115)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_4"></a>|@therve set owner to @exarkun|
|-|-|

+1!

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set status to `closed`|
|-|-|

(In [27543]) Merge lore-lint-getattribute-fixes-4115

Author: exarkun
Reviewer: therve
Fixes: [#4115](https://github.com/twisted/twisted/issues/4115)

Add a test for one of the lore lint checks related to anchor tags and
fix the implementation so that it works with minidom's getAttribute
method (which only accepts one argument).  Also delete another of the
anchor tag lint checks, the one which looked for license text being
included in source listings, as it seems somewhat useless.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
